src/webview2/mod.rs

```diff
@@ -1190,22 +1192,17 @@ impl InnerWebView {
       }];
       unsafe { MapWindowPoints(self.hwnd, *self.parent.borrow(), position_point) };
 
-      bounds.position = LogicalPosition::new(position_point[0].x, position_point[0].y).into();
-      bounds.size = LogicalSize::new(
-        (rect.right - rect.left) as u32,
-        (rect.bottom - rect.top) as u32,
-      )
-      .into();
+      bounds.position = PhysicalPosition::new(position_point[0].x, position_point[0].y)
+        .to_logical::<f64>(scale_factor)
```

Comment:
no need to convert to logical

Reply:
@amrbashir 
https://github.com/tauri-apps/tauri/issues/10053

Have you seen this issue?
This is caused by not converting to logical.

Reply:
the field `position` and `size` in `Rect` are of type `Position` and `Size` which can be constructed directly from `Physical` or `Logical` type, it is the responsibility of the user to convert the position/size later to their preferred variant.

Reply:
I think Amr means Rect can take PhysicalPosition dierctly, and we can convert it later when needed.

Reply:
you can do

```rs
Rect {
	position: PhysicalPositon::new().into(),
}
```
as well as:
```rs
Rect {
	position: LogicalPosition::new().into(),
}
```

and the user of `.bounds()` function can convert to their needed types:
```rs
let bounds = webview.bounds()?;
let position = bounds.position.to_logical(scale_factor);
```

Reply:
Is this what you mean?
```
bounds.position = PhysicalPosition::new(position_point[0].x, position_point[0].y).into();
```

The original code is wrong because it uses logical, right?
```
bounds.position = LogicalPosition::new(position_point[0].x, position_point[0].y).into();
```

Reply:
yeah this is what I mean

Reply:
I still haven't tested this so I am not sure yet if the correct value should be `LogicalPosition` or `PhysicalPosition` and the bug could in `set_bounds` not `bounds` where it should be passing logical values to win32 APIs instead of physical.

Reply:
I got a bit confused by the definition of `Logical` lol

- (set) `SetWindowPos` in `set_bound` should be `Logical`, meaning it should have DPI awareness.
- (get) The position we get from `MapWindowPoints` should be `Physical` since our app is DPI unawareness.
- So the `LogicalPosition` for  `SetWindowPos` should be the `PhysicalPosition` from `MapWindowPoints` * `dpi_scale_factor`, which gets a larger value if the DPI is higher.

It looks different from macOS's implementation because macOS will get a smaller coordinate if the DPI is higher.

Reply:
I'm not sure what we get from WIN32 API is already scaled now :cry: , but I think we should use `Logical` in `set_bounds`